providers/certs.rb

```diff
@@ -190,7 +197,9 @@
         # Import into the keystore
         keytool -importkeystore -destkeystore keystore.jks -srckeystore cert_and_key.p12 -srcstoretype PKCS12 -alias internal -srcstorepass #{node['hopsworks']['master']['password']} -deststorepass #{node['hopsworks']['master']['password']} -destkeypass #{node['hopsworks']['master']['password']}
 
-         keytool -import -noprompt -alias internal -file #{node['certs']['dir']}/certs/ca.cert.pem -keystore cacerts.jks -storepass #{node['hopsworks']['master']['password']}
+        keytool -import -noprompt -alias internal -file #{node['certs']['dir']}/certs/ca.cert.pem -keystore cacerts.jks -storepass #{node['hopsworks']['master']['password']}
+        
+        keytool -import -noprompt -alias digicertglobalrootg2 -file /tmp/DigiCertGlobalRootG2.crt -keystore cacerts.jks -storepass #{node['hopsworks']['master']['password']}
```

Comment:
Same as above, create a separate `action` and call it from `hopsworks::default`?